internal/wazeroir/operations.go

```diff
@@ -1393,36 +1379,24 @@ func (OperationStore32) Kind() OperationKind {
 	return OperationKindStore32
 }
 
-// OperationMemorySize implements Operation.
+// NewOperationMemorySize is a constructor for OperationNullary with Kind OperationKindMemorySize.
 //
 // This corresponds to wasm.OpcodeMemorySize.
 //
 // The engines are expected to push the current page size of the memory onto the stack.
-type OperationMemorySize struct{}
-
-// String implements fmt.Stringer.
-func (o OperationMemorySize) String() string { return o.Kind().String() }
-
-// Kind implements Operation.Kind.
-func (OperationMemorySize) Kind() OperationKind {
-	return OperationKindMemorySize
+func NewOperationMemorySize() OperationNullary {
+	return OperationNullary{kind: OperationKindMemorySize}
 }
 
-// OperationMemoryGrow implements Operation.
-type OperationMemoryGrow struct{ Alignment uint64 }
```

Comment:
MemoryGrow is not really nullary but we discard the `Alignment` field everywhere